lib/discourse_chat/provider/telegram/telegram_command_controller.rb

```diff
@@ -29,7 +29,7 @@ def command
 
         DiscourseChat::Provider::TelegramProvider.sendMessage(message)
 
-      elsif params.key?('channel_post') && params['channel_post']['text'].include?('/getchatid')
+      elsif params.key?('channel_post') && (params['channel_post']['text'] || "").include?('/getchatid')
```

Comment:
A more concise way to write this could be:

```suggestion
      elsif params.dig('channel_post', 'text')&.include?('/getchatid')
```

(love me some code golf 😜)

Reply:
Ta.  Rebased.